.gitignore

```diff
@@ -49,3 +49,6 @@ repo/
 flatpak-build/
 .flatpak-builder/
 *.flatpak
+
+# LUA build
+lua-5.3.5/
```

Comment:
Please use `lua-*/` instead, as we risk missing this at a version bump...

Reply:
did that :)